apps/prairielearn/python/prairielearn.py

```diff
@@ -1553,7 +1553,7 @@ def latex_from_2darray(
     return "".join(rv)
 
 
-def is_correct_ndarray2D_dd(a_sub, a_tru, digits=2):
+def is_correct_ndarray2d_dd(a_sub, a_tru, digits=2):
```

Comment:
Note this is potentially breaking for client code, though the usage of these functions is unlikely.

Reply:
Will confirm with @mwest1066 if this is breaking.

Reply:
This is breaking for exactly one course.

Reply:
I think the simplest is just to revert it back and do a ruff ignore.